doc/nrf/gs_programming.rst

```diff
@@ -91,7 +91,7 @@ You can download it from the following links:
       a. Select your project in the Project Explorer.
       #. From the menu, select **Build -> Build Solution**.
       #. When the multi-image build completes, you can flash the sample to a connected board.
-         To do this, select **Target -> Download File -> Download Intel Hex File**.
+         To do this, select **Target -> Download zephyr/merged.hex**.
          Navigate to the ``zephyr`` folder in your build directory and choose ``merged.hex``.
```

Comment:
I would rephrase the text from line: 77-96: into:
```
 To build and flash an application:
      a. Select your project in the Project Explorer.
      #. From the menu, select **Build -> Build Solution**.
      #. When the build completes, you can flash the sample to a connected board.
         To do this, select
              **Target -> Download zephyr/zephyr.elf**. (single image application)
              **Target -> Download zephyr/merged.hex**. (multi image application)

      .. note::
	   Alternatively, choose the **Build and Debug** option.
	   **Build and Debug** will build the application and flash it when
the build completes.
```

Reply:
thanks @tejlmand  , please verify and approve

Just to double-check  - the new SES version is 4.18?